app/utils/ExtractMeta.js

```diff
@@ -57,6 +58,26 @@ export default function extractMeta(chain_data, rp) {
         } else {
             addSiteMeta(metas);
         }
+    } else if (rp.accountname) { // user profile root
+        const account = chain_data.accounts[rp.accountname];
+        let {name, about, profile_image} = normalizeProfile(account);
+        if(name == null) name = account.name;
+        if(about == null) about = " Steemit";
```

Comment:
`about = " Steemit";` looks weird, better to just leave it blank? Or maybe add some other bit of into?